### PR TITLE
[Database] list all clinic_types

### PR DESCRIPTION
**Pull request by ghost**
## Description
- Create model ClinicType
- Create table clinic_types
- Create clinic_types index view
- Show list clinic_types

## Migrations
YES

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -11,8 +11,7 @@
 |
 */
 
-Route::group(['prefix' => 'admin'], function() {
-    Route::get('/dashboard', function() {
-        return view('admin.dashboard');
-    });
+Route::group(['prefix' => 'admin', 'as' => 'admin.'], function() {
```
Comment (at-quantran2) [inline routes/web.php]: 1 space sau function

**Comment by at-quantran2** [comment]
LGTM

**Comment by at-longtran** [comment]
LGTM

File: app/Http/Controllers/ClinicTypeController.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Http\Controllers;
```
Comment (at-vienle) [inline app/Http/Controllers/ClinicTypeController.php]: move to `Admin` namespace

**Review by ghost — CHANGES_REQUESTED** [review]
check my cmt

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**